google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITBlobReadMaskTest.java

```diff
@@ -201,7 +201,9 @@ public ImmutableList<?> parameters() {
               new Args<>(BlobField.UPDATED, LazyAssertion.equal()),
               new Args<>(
                   BlobField.RETENTION,
-                  LazyAssertion.skip("TODO: jesse fill in buganizer bug here")));
+                  LazyAssertion.skip("TODO: jesse fill in buganizer bug here")),
```

Comment:
I understand this code already exists; just wondering why is Retention field mask test skipped?

Reply:
This is related to object retention being JSON only. This will be removed once it's in the grpc api as well